packages/ods/src/components/tabs/src/components/ods-tabs/ods-tabs.scss

```diff
@@ -5,7 +5,15 @@
 
   display: inline-flex;
   flex-flow: row;
-  padding: var(--ods-outline-width);
-  width: inherit;
-  overflow-x: auto;
+  padding: var(--ods-outline-width) 0;
+  width: 100%;
+  overflow-y: auto;
```

Comment:
`overflow-x` we want an horizontal scroll (it won't impact your tab rendering with the wrapper)